src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1631,30 +1631,56 @@ void C2_MacroAssembler::minmax_fp_v(VectorRegister dst, VectorRegister src1, Vec
   vfadd_vv(dst, src2, src2, Assembler::v0_t);
 }
 
+// Set dst to NaN if any NaN input.
+void C2_MacroAssembler::minmax_fp_masked_v(VectorRegister dst_src1, VectorRegister src2,
```

Comment:
Better to break down `dst_src1` into two seperate operands, i.e., `dst` and `src1`.

Reply:
Fixed.